backend/core/admin_database.py

```
@@ -226,41 +295,616 @@ def _ensure_default_admin_user(self, cursor):
+                else:
+                    cursor.execute(
+                        """
+                        SELECT id, name, display_name, description, icon, sort_order, is_active, created_at, updated_at
+                        FROM followup_categories
+                        ORDER BY sort_order, display_name  
+                        """
+                    )
+
+                return [dict(row) for row in cursor.fetchall()]
+        except Exception as e:
+            logger.error(f"Error getting followup categories: {str(e)}", exc_info=True)
```

Ruff (0.12.2)

Use explicit conversion flag

Replace with conversion flag

(RUF010)